src/cdomains/mHP.ml

```diff
@@ -92,3 +92,8 @@ let may_happen_in_parallel one two =
     else
       true
   | _ -> true
+
+let may_be_non_unique_thread mhp =
+  match mhp.tid with
+  | `Lifted tid -> not (TID.is_unique tid)
```

Comment:
Why is this `may_be` when the underlying thread ID is directly known?

Reply:
Just `is_non_unique_thread` I guess. Or maybe even avoiding the flip and `is_unique_thread`.